ENIGMAsystem/SHELL/Universal_System/GAME_GLOBALS.h

```diff
@@ -29,27 +29,16 @@
 GM Global variables
 ******************/
 
-#ifndef __GAME_GLOBALS_H
-#define __GAME_GLOBALS_H
+#ifndef ENIGMA_GAME_GLOBALS_H
+#define ENIGMA_GAME_GLOBALS_H
 
-bool argument_relative=false;
```

Comment:
Moving this into enigma_user will break, and has broken, drag and drop:
https://github.com/enigma-dev/enigma-dev/pull/1128/files#r161388840